cmd/gateway-conformance/main.go

```diff
@@ -85,23 +85,18 @@ func main() {
 						Value:       "",
 						Destination: &jsonOutput,
 					},
-					&cli.BoolFlag{
-						Name:        "subdomain-gateway-spec",
-						Aliases:     []string{"subdomain-gateway", "subdomain"},
-						Usage:       "Whether the gateway implements the subdomain gateway spec",
-						Value:       true,
-						Destination: &subdomainGatewaySpec,
+					&cli.StringFlag{
+						Name:        "specs",
+						Usage:       "A comma-separated list of specs to test",
+						Value:       "",
+						Destination: &specs,
```

Comment:
I thought about it and I think I'd prefer an interface that people can get used to. If we have separate flag for each spec (as it was here before), then we'd have to change that interface every time a new spec is added to the tests. I think it'd become quite overwhelming with the amount of different specs we might want to support. Also, it'd be harder to maintain as we'd have to remember to update the lists in all these places. In my opinion a single (or three) flags that take a list of specs will be easier to manage in a long run.

Reply:
It might be useful to add a command for listing all the specs!